tests/unit_test/gmail_data/get_gmail_data_test.py

```diff
@@ -0,0 +1,219 @@
+from unittest.mock import MagicMock
+import pytest
+
+from data_pipeline.gmail_data.get_gmail_data import (
+    get_dataframe_for_label_response,
+    get_dataframe_for_thread_response,
+    get_dataframe_for_history_response,
+    iter_link_message_thread_ids,
+    iter_gmail_history,
+    get_label_list,
+    get_one_thread
+)
+
+USER_ID1 = 'user_id1'
+IMPORTED_TIMESTAMP = '2021-04-16'
+
+THREAD_ID1 = 'thread_id1'
+
+START_ID1 = 'start_id1'
+
+
+@pytest.fixture(name='gmail_service_mock')
+def _gmail_service_mock() -> MagicMock:
+    return MagicMock(name='service')
+
+
+@pytest.fixture(name='gmail_label_list_mock')
+def _gmail_label_list_mock(gmail_service_mock: MagicMock) -> MagicMock:
+    users_mock = gmail_service_mock.users
```

Comment:
minor: just an idea, you could also extract users to it's own fixture as it is used in a number of fixtures.